### PR TITLE
Add renderer kwarg to CodeMirrorTextarea.renderer for Django 2.1+ compatibility

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

File: codemirror/widgets.py [hunk 1]
```diff
@@ -159,13 +159,14 @@ def __init__(
...
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
         u"""Render CodeMirrorTextarea"""
         if self.js_var_format is not None:
             js_var_bit = 'var %s = ' % (self.js_var_format % name)
         else:
             js_var_bit = ''
-        output = [super(CodeMirrorTextarea, self).render(name, value, attrs),
+        output = [super(CodeMirrorTextarea, self).render(
+                        name, value, attrs, renderer),
```
Comment (lambdalisue) [inline codemirror/widgets.py]: I guess this code does not work with the old Django because of the number of arguments are changed.

Comment (inducer) [reply codemirror/widgets.py]: Would you like me to insert a version query? (As in, an `if` statement with both code paths)

Comment (lambdalisue) [reply codemirror/widgets.py]: Yes please 👍

Comment (inducer) [reply codemirror/widgets.py]: Done.

**Comment by lambdalisue** [comment]
Sorry for late and thank you 👍

**Comment by lambdalisue** [comment]
Released as 0.5.0 https://pypi.org/project/django-codemirror-widget

**Comment by inducer** [comment]
Thanks!

**Pull request merged**